kod/object/passive/spell/persench.kod

```diff
@@ -164,6 +167,15 @@ messages:
 
       local oTarget, lType;
 
+      if NOT bItemCast
+         AND Send(self,@CanBeReserved)
```

Comment:
And if @CanBeReserved is called on self here (and always just returns the classvar) we can change this line to AND vbCanBeReserved